frontend/views/containers/proposals/RuleAddMember.vue

```diff
@@ -1,20 +1,76 @@
 <template lang="pug">
 proposal-template(
-  :subTitle="L('Change Rule to Add Member')"
-  :onSubmit='handleSubmit'
-) I step to change rule of add member it's on the way.
+  title= 'Change Rule to Add Member'
+  footer='According to your voting rules, 8 out of 10 members will have to agree with this.'
+  :disabled='$v.form.$invalid || ($v.steps[config.steps[currentStep]] && $v.steps[config.steps[currentStep]].$invalid)'
+  :maxSteps='config.steps.length'
+  :currentStep.sync='currentStep'
+  @submit='submit'
+)
+
+  .field(v-if='currentStep === 0' key='0')
+    label.label I step to change rule of add member it's on the way.
+
+  .field(v-if='currentStep === 1' key='1')
+    label.label
+      i18n Why are you proposing this change?
+
+    textarea.textarea(
+      name='changeReason'
+      ref='purpose'
+      placeholder='The reason why I\' propositiong this change is...'
```

Comment:
`The reason why I\'m ...`? (missing m). Note that this PR is missing many i18n tags / L functions. They need to be added everywhere there is text.